### PR TITLE
Unit test

### PR DESCRIPTION
**Pull request by momotsow**
### In this milestone we have: 
- Added unit tests for all the methods in all the classes:
  - `Book`
  - `Person`
  - `Student`
  - `Teacher`
  - `Classroom`
  - `Rental`

### CODE REVIEW

**Review by rotimiazeez — CHANGES_REQUESTED** [review]
# Hi TEAM 👍🏽,
## Good job so far!
### To Highlight 👇🏽
- No Linter error ✔️ 
- Good PR description 👍🏽 
- Good commit message(s) 😉
- Descriptive README 📖
You still need to work on some issues to prepare your project for the final evaluation, but you are almost there!


## Suggested changes
### Please ensure the license link you provided is valid 👍🏽
Screenshot from 2023-02-10 19-41-49
### Kindly ensure a fair distribution of workload between your partner 👍🏽 

_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you consider them as they can improve your code._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag (@rotimiazeez) me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid, unless it is requested otherwise._**

**Comment by momotsow** [comment]
@rotimiazeez Thank you for the review. We have fixed the Licence link and our workload was divided equally it is just a matter of commits.
We will in future commit equally.

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2023 Hevar Hoshang Ahmed
+
```
Comment (TadesseDev) [inline LICENSE:4]: - [Optionl] Hello team, Good job making the changes, but while you do so, please remember this is a group project, and as such the License file should be granted by the name of both collaborators. For the time being I can see only `Hevar Hoshang Ahmed`.

**Review by TadesseDev — APPROVED** [review]
### Hi  Team
### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
giphy (1)



## Highlights
 - [ ] Good job making the changes requested by the previous code reviewer. ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**